app/src/main/java/com/example/tasky/agenda/presentation/agenda_detail/AgendaDetailViewModel.kt

```diff
@@ -92,6 +103,59 @@ class AgendaDetailViewModel(
                     )
                 }
             }
+            is AgendaDetailAction.OnPhotoSelected -> {
+                viewModelScope.launch(io) {
```

Comment:
I would follow Philipp's suggestion here `Dispatchers.Default` is more suitable for this job.